### PR TITLE
update to pcidevices chart

### PR DESCRIPTION
**Pull request by ibrokethecloud**
changes to pcidevices chart to support additional webhook.

### CODE REVIEW

File: charts/harvester-pcidevices-controller/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.0.0-dev
+version: 0.1.0-dev
```
Comment (tlehman) [inline charts/harvester-pcidevices-controller/Chart.yaml:18]: pcidevices is at version 0.2.0, is this a different version? And if so, should I update the pcidevices repo to reflect this?

Comment (ibrokethecloud) [reply charts/harvester-pcidevices-controller/Chart.yaml:18]: this is just a baseline chart.. the actual versions are immaterial. the next PR into `release` will need to valid build version.

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Review by bk201 — APPROVED** [review]

**Comment by bk201** [comment]
@ibrokethecloud Could help backport to `release` branch once pcidevices is ready and tagged. Thanks.

**Comment by ibrokethecloud** [comment]
@bk201 already on it.

**Pull request merged**